tests/tests_geomstats/test_minkowski.py

```diff
@@ -259,6 +277,52 @@ def parallel_transport_bvp_is_isometry_test_data(self):
                 atol=gs.atol * 1000,
             )
 
+        def dist_is_symmetric_test_data(self):
+            return self._dist_is_symmetric_test_data(
+                self.metric_args_list,
+                self.space_list,
+                self.n_points_a_list,
+                self.n_points_b_list,
+            )
+
+        def dist_is_positive_test_data(self):
+            return self._dist_is_positive_test_data(
+                self.metric_args_list,
+                self.space_list,
+                self.n_points_a_list,
+                self.n_points_b_list,
+            )
+
+        def squared_dist_is_positive_test_data(self):
+            return self._squared_dist_is_positive_test_data(
+                self.metric_args_list,
+                self.space_list,
+                self.n_points_a_list,
+                self.n_points_b_list,
+                is_positive_atol=gs.atol * 10000,
```

Comment:
Minkowski is a pseudo-Riemannian metric (it is actually the only one in the library), thus the squared dist can be negative, I would suggest removing this test (ignoring, as opposed to skipping).